torchgeo/datamodules/usavars.py

```diff
@@ -33,18 +31,28 @@ def __init__(
         Args:
             root_dir: The root argument passed to the USAVars Dataset classes
             labels: The labels argument passed to the USAVars Dataset classes
-            transforms: a function/transform that takes input sample and its target as
-                            entry and returns a transformed version
             batch_size: The batch size to use in all created DataLoaders
             num_workers: The number of workers to use in all created DataLoaders
         """
         super().__init__()
         self.root_dir = root_dir
         self.labels = labels
-        self.transforms = transforms
         self.batch_size = batch_size
         self.num_workers = num_workers
 
+    def preprocess(self, sample: Dict[str, Any]) -> Dict[str, Any]:
+        """Transform a single sample from the Dataset.
+
+        Args:
+            sample: dictionary containing image
+
+        Returns:
+            preprocessed sample
+        """
+        sample["image"] = sample["image"].float()
+        sample["image"] /= 255.0
```

Comment:
It seems like we're somewhat arbitrarily deciding whether to normalize to [0, 1] or [-1, 1]. Not sure if there is any advantage to one or the other.

Reply:
Yep!

Reply:
I wouldn't mind arbitrarily being consistent and do [0,1] normalization for everything.

Reply:
I'm not sure if this needs to be solved in this PR, just something to think about. We should probably try both and see if it makes any difference. It may be different on a case-by-case basis.